platform/security/platform-security-core-api/src/main/java/ddf/security/samlp/SamlProtocol.java

```diff
@@ -216,6 +216,7 @@ public static Binding from(String value) {
         }
 
         public boolean isEqual(String uri) {
+            // TODO: 10/13/16 Address checkstyle warning of uri hiding a field 
```

Comment:
This happens all over ddf; it's not terribly important that we address them.  We can remove this comment